test/e2e/suite/nodeadm_test.go

```diff
@@ -555,3 +555,13 @@ func newE2EAWSSession(region string) (*session.Session, error) {
 	}
 	return sess, nil
 }
+
+func instanceType(region string, arch string) string {
```

Comment:
Why can't the OSs just call this method? We are now exposing more details, making the interface more complicated

Reply:
I thought the method Arch() might be better to be set on an interface called `NodeadmOS` than `InstanceType`. I might be wrong about this, anyway I have fixed this.